src/Network/HTTPClient/Capability/ICanSendHttpRequests.php

```diff
@@ -50,72 +51,75 @@ public function fetch(string $url, int $timeout = 0, string $accept_content = ''
 	 * all the information collected during the fetch.
 	 *
 	 * @param string $url             URL to fetch
-	 * @param int    $timeout         Timeout in seconds, default system config value or 60 seconds
 	 * @param string $accept_content  supply Accept: header with 'accept_content' as the value
+	 * @param int    $timeout         Timeout in seconds, default system config value or 60 seconds
 	 * @param string $cookiejar       Path to cookie jar file
 	 *
 	 * @return ICanHandleHttpResponses With all relevant information, 'body' contains the actual fetched content.
 	 */
-	public function fetchFull(string $url, int $timeout = 0, string $accept_content = '', string $cookiejar = ''): ICanHandleHttpResponses;
+	public function fetchFull(string $url, string $accept_content = HttpClientAccept::DEFAULT, int $timeout = 0, string $cookiejar = ''): ICanHandleHttpResponses;
 
 	/**
 	 * Send a HEAD to a URL.
 	 *
-	 * @param string $url        URL to fetch
-	 * @param array  $opts       (optional parameters) associative array with:
-	 *                           'accept_content' => (string array) supply Accept: header with 'accept_content' as the value
-	 *                           'timeout' => int Timeout in seconds, default system config value or 60 seconds
-	 *                           'cookiejar' => path to cookie jar file
-	 *                           'header' => header array
+	 * @param string $url            URL to fetch
+	 * @param string $accept_content supply Accept: header with 'accept_content' as the value
+	 * @param array  $opts           (optional parameters) associative array with:
+	 *                                'accept_content' => (string array) supply Accept: header with 'accept_content' as the value (overrides default parameter)
```

Comment:
Redundant with the explicit parameter.
```suggestion
```

Reply:
As discussed with @annando , we want to add this explicit option either for explicit set for `head` and `post` again, or because we already set it for the HTTP Header signing process

Reply:
We have got different signing processes and these are already separated in discrete functions. I don't think that we should merge this stuff.